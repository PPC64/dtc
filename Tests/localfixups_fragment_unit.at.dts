/dts-v1/;
/plugin/;

/ {
    fragment@1 {
        target = <&foo>;
        __overlay__ {
            f1_label: f1_node {
            };
        };
    };

    fragment@2 {
        target = <&bar>;
        __overlay__ {
            ref-0 = <&f1_label>;
        };
    };
};
