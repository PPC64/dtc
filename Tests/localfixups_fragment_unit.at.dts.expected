/dts-v1/;

/  {

	fragment@1 {

		target = <0xdeadbeef>;
		__overlay__ {

			f1_node {

				phandle = <0x1>;
			};
		};
	};
	fragment@2 {

		target = <0xdeadbeef>;
		__overlay__ {

			ref-0 = <0x1>;
		};
	};
	__symbols__ {

		f1_label = "/fragment@1/__overlay__/f1_node";
	};
	__fixups__ {

		foo = "/fragment@1:target:0";
		bar = "/fragment@2:target:0";
	};
	__local_fixups__ {

		fragment@2 {

			__overlay__ {

				ref-0 = <0x0>;
			};
		};
	};
};
